code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -0,0 +1,400 @@
+/mob/living/simple_animal/bot/honkbot
+	name = "\improper Honkbot"
+	desc = "A little robot. It looks happy with its bike horn."
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "honkbot0"
+	density = FALSE
+	anchored = FALSE
+	health = 25
+	maxHealth = 25
+	damage_coeff = list(BRUTE = 0.9, BURN = 1.0, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+
+	radio_key = /obj/item/device/encryptionkey/headset_service //doesn't have security key
+	radio_channel = "Service" //Service
+	bot_type = HONK_BOT
+	model = "Honkbot"
+	bot_core_type = /obj/machinery/bot_core/honkbot
+	window_id = "autohonk"
+	window_name = "Honkomatic Bike Horn Unit v1.0.2"
+	//allow_pai = 1 //Damn right we'll pAI these
+	data_hud_type = DATA_HUD_SECURITY_ADVANCED // show jobs
```

Comment:
Use `DATA_HUD_SECURITY_BASIC` for jobs-only display.

Reply:
Why does it need to see jobs?